raiden/channel/netting_channel.py

```diff
@@ -276,16 +277,19 @@ def get_settle_expiration(self, block_number):
 
         return blocks_until_settlement
 
-    def handle_closed(self, block_number):  # pylint: disable=unused-argument
+    def handle_closed(self, block_number, closing_address):  # pylint: disable=unused-argument
```

Comment:
Why are you adding this argument ,`closing_address`, here? It does not seem to be used.

Reply:
Check [line 801](https://github.com/raiden-network/raiden/pull/1003/commits/9726e4da7c905117f6dc12f79a18b92d99ddf60b#diff-263eac5cb6d265279d2898e4c2bb953fR801)

Reply:
Hmm how come I missed that. Perhaps I was searching for a green diff showing the `closing_address` but it's just a variable that was local to the function before. Thanks.